Sources/Momento/messages/responses/topics/TopicSubscriptionItem.swift

```diff
@@ -6,20 +6,28 @@ public enum TopicSubscriptionItemResponse {
     case error(TopicSubscriptionItemError)
 }
 
-public class TopicSubscriptionItemText {
+public class TopicSubscriptionItemText: CustomStringConvertible {
     public let value: String
     
     init(value: String) {
         self.value = value
     }
+    
+    public var description: String {
+        return "[\(type(of: self))] Value: \(self.value)"
+    }
 }
 
-public class TopicSubscriptionItemBinary {
+public class TopicSubscriptionItemBinary: CustomStringConvertible {
     public let value: Data
     
     init(value: Data) {
         self.value = value
     }
+    
+    public var description: String {
+        return "[\(type(of: self))] Value: \(self.value)"
```

Comment:
We should decode this before displaying it:
```suggestion
        return "[\(type(of: self))] Value: \(String(decoding: self.value, as: UTF8.self))"
```